Xamarin.Forms.Platform.Tizen/Renderers/SwipeViewRenderer.cs

```diff
@@ -0,0 +1,435 @@
+﻿using System;
+using System.Threading.Tasks;
+using ElmSharp;
+
+namespace Xamarin.Forms.Platform.Tizen
+{
+	enum SwipeDrawerState
+	{
+		Opend,
+		Closed
+	}
+
+	public class SwipeViewRenderer : LayoutRenderer
+	{
+		static readonly double SwipeItemWidth = 100;
```

Comment:
Please check again whether it is a valid size even wearable that has a relatively small screen.

Reply:
Will change to 100 px not dp